packages/backend/src/nest/app.module.ts

```diff
@@ -153,6 +153,11 @@ export class AppModule {
           useFactory: (baseDir: string) => path.join(baseDir, 'backendDB'),
           inject: [QUIET_DIR],
         },
+        {
```

Comment:
This was added because I was experimenting with using Level for the Libp2p datastore.  Right now I'm using a memory store but it might be worth doing a persisted store in the future.